metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -438,6 +438,12 @@ public void add(final Value value) {
             }
         }
 
+        /* package-private */ void add(final int index, final Value value) {
+            if (!isNull(value)) {
+                list.add(index, value);
```

Comment:
Does this not require the `appendToPath` behaviour from `add(Value, boolean)`?

Reply:
Yes, right, thanks, added in https://github.com/metafacture/metafacture-fix/pull/368/commits/d00e59a173ba38c8614f65ca4b707523c73b7eae.

Reply:
Great. Now we could also implement `add(Value, boolean)` in terms of `add(int, Value, boolean)` (with `index = list.size()`). But that's certainly a minor detail.

Reply:
Ah, yes, nice, done: https://github.com/metafacture/metafacture-fix/pull/368/commits/fe84442c0f37983996ef02c31b660bc5235dce12